analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/LooseFilePermissionsTest.cs

```diff
@@ -19,44 +19,53 @@
  */
 
 using SonarAnalyzer.Common;
```

Comment:
I think we should add a test case with raw string literals for the "identity" parameter.